tools/roc/application.e

```diff
@@ -0,0 +1,287 @@
+note
+	description: "[
+			roc tool: install modules into an existing CMS Application.
+			
+			roc install [--module|-m <MODULE_PATH>] [(--dir|-d <CMS_PATH>) | <MODULE_NAME>]
+
+			install:  Install a given module to the corresponding cms application
+			--module|-m: module path or current directory if is not defined.
+			--dir|-d cms application path or current directory if is not defined
+
+			Running the command will copy to the CMS Application site/modules the following artifacts if the current module provide them.
+
+				config
+				scripts
+				themes
+			running
+			roc instal blog
+			will look for a module blog in the modules directory starting at the current directory.
+	]"
+	date: "$Date$"
+	revision: "$Revision$"
+
+class
+	APPLICATION
+
+inherit
+
+	SHARED_EXECUTION_ENVIRONMENT
+
+	ARGUMENTS
+
+create
+	make
+
+feature {NONE} -- Initialization
+
+	make
+			-- Initialize tool.
+		do
+				-- TODO add support to other commands.
+			if argument_count = 0 then
+				print ("Use: roc install [--module|-m <MODULE_PATH>] [(--dir|-d <CMS_PATH>) | <MODULE_NAME>]")
+			elseif argument (1).starts_with ("install") then
+				execute_install
+			else
+				print ("Wrong command")
```

Comment:
precise which command is wrong .. this will give a clue to the user where is his error.